src/olympia/addons/tests/test_models.py

```diff
@@ -329,13 +325,12 @@ def test_filter_for_many_to_many(self):
         collection = self.addon.collections.first()
         assert collection.addons.get() == self.addon
 
-        # Addon shouldn't be listed in collection.addons if it's deleted or
-        # unlisted.
+        # Addon shouldn't be listed in collection.addons if it's deleted.
 
         # Unlisted.
         self.addon.update(is_listed=False)
         collection = Collection.objects.get(pk=collection.pk)
-        assert collection.addons.count() == 0
+        assert collection.addons.get() == self.addon
```

Comment:
It was empty because it excluded unlisted add-ons. It does not anymore (it doesn't need to filter because the views should filter on status when needed)